packages/loader/driver-utils/src/test/summaryCompresssionTester.spec.ts

```diff
@@ -131,7 +135,7 @@ class InternalTestStorage implements IDocumentStorageService {
 	}
 	async readBlob(id: string): Promise<ArrayBufferLike> {
 		return id === misotestid
-			? new TextEncoder().encode(abcContent)
+			? (new TextEncoder().encode(abcContent) as unknown as ArrayBufferLike)
```

Comment:
This returns a Uint8Array right ( https://developer.mozilla.org/en-US/docs/Web/API/TextEncoder/encode ). If those are suable as an ArrayBufferLike, we should not need to do a very unsafe cast here.

It seems like such an assignment is not allowed?

This change also seems different from the others, and should not be required in this PR.

Mayne there was some platforms specific buffer typing which was highlighted in IntelliSense but not a real compile error?

Reply:
Yup. Fixed.